docs/topics/proc_accessing-jolokia-using-hawtio-on-openshift.adoc

```diff
@@ -4,27 +4,68 @@
 link:https://jolokia.org/documentation.html[Jolokia^] is a built-in lightweight solution for accessing JMX (Java Management Extension) metrics over HTTP on OpenShift.
 It allows you to access CPU, storage, and memory usage data collected by JMX over an HTTP bridge using a REST interface and JSON-formatted message payloads, making it suitable for monitoring cloud applications, due to its comparably high speed and low resource-requirements.
 
-
-All Fabric8 container images expose port `8778` by default on the host of your {OpenShiftLocal}.
-
 [accessing-jvm-metrics-using-hawtio-console]
 = Accessing Jolokia metrics using the hawt.io console
 
 For Java-based applications, the OpenShift Web console provides the integrated link:https://docs.openshift.com/container-platform/3.6/architecture/infrastructure_components/web_console.html#jvm-console[_hawt.io_ console^] that collects and displays all relevant metrics output by the JVM running your application.
 
 .Prerequistes
 
-* the `oc` client uthenticated
+* the `oc` client authenticated
 * a Java-based application container running in a project on OpenShift
-* port `8778` exposed in the container image (exposed by default in Fabric8 application images)
 * latest link:https://github.com/jboss-container-images/redhat-openjdk-18-openshift-image/blob/openjdk18-dev/image.yaml[JDK 1.8.0 image^]
 
 .Procedure
 
-. Log into the OpenShift Web console.
+. Obtain the name of the pod running your application:
++
+[source,bash,options="nowrap",subs="attributes+"]
+--
+oc get pods
```

Comment:
So if you below edit the deployment config, shouldn't you list the deployment configs here? `oc get dc`

Reply:
@Ladicek, yes, that should by all means be mentioned. I'll add it in, thank you for catching this:)

Reply:
@Ladicek I've added the step to list the deployment config using `oc get dc`

Reply:
I think you don't really need to look for the pods. After changing the dc, the pods will have different names anyway. I'd personally get rid of `oc get pods` and just did `oc get dc` (_not_ `oc get dc app-name`). That will tell me the names of all deployment configs and from there, I select the one which I will `oc edit`.

It's not that big of a deal really -- pods managed by a dc are always named after the dc. Your call :-)